spec/DecodeEventsSpec.hs

```diff
@@ -5653,6 +5654,129 @@ pushEventFixture = PushEvent
           }
     }
 
+pushEventFixtureWithoutSender :: PushEvent
+pushEventFixtureWithoutSender =
+  PushEvent
+    { evPushRef = "refs/heads/gh-readonly-queue/main/pr-3072-287d30540ac5a1acc57d0bdc138fa81dee315f48",
+      evPushHeadSha = Just "0000000000000000000000000000000000000000",
+      evPushBeforeSha = Just "a6bd77bbe77ee6ae8f9686621b192d70fcf83e2b",
+      evPushCreated = False,
+      evPushDeleted = True,
+      evPushForced = False,
+      evPushBaseRef = Nothing,
+      evPushCompareUrl = URL "https://github.com/some-organization/infra-core/compare/a6bd77bbe77e...000000000000",
+      evPushCommits = Just mempty,
```

Comment:
Please see: https://github.com/cuedo/github-webhooks/actions/runs/3680596117/jobs/6226368854#step:4:172

Reply:
7.8...
I'm going to fix it now, but are you sure you need a 7 year old compiler?

Reply:
The fix is simple, so let's keep compatibility for now. I know several large companies are using this and I'm not sure which compilers they are using it with.

Reply:
Fair enough, appreciate the commitment.